markets/perps-market/subgraph/tests/handleAccountLiquidated.ts

```diff
@@ -1,15 +1,24 @@
-import { assert } from 'matchstick-as';
-import { Address, BigInt, ethereum } from '@graphprotocol/graph-ts';
+import { assert, log } from 'matchstick-as';
 import { handleAccountLiquidated } from '../optimism-goerli';
 import { createAccountLiquidatedEvent } from './event-factories/createAccountLiquidatedEvent';
 
 export default function test(): void {
-  const accountLiquidatedEvent = createAccountLiquidatedEvent(1, 123, true, 10_000, 10);
-
-  handleAccountLiquidated(accountLiquidatedEvent);
+  assert.entityCount('AccountLiquidated', 0);
 
+  log.info('Should create a new record for the liquidation', []);
+  handleAccountLiquidated(createAccountLiquidatedEvent(1, 123, true, 10_000, 10));
+  assert.entityCount('AccountLiquidated', 1);
   assert.fieldEquals('AccountLiquidated', '1-10', 'accountId', '1');
   assert.fieldEquals('AccountLiquidated', '1-10', 'timestamp', '10000');
   assert.fieldEquals('AccountLiquidated', '1-10', 'liquidationReward', '123');
   assert.fieldEquals('AccountLiquidated', '1-10', 'fullyLiquidated', 'true');
+
+  // TODO: can we have 2 separate liquidation events in the same block? Potentially yes
+  log.info('Should create another record for the liquidation of the same account', []);
```

Comment:
I dont think so, but not 100% sure.. I have a vague memory that when an account has been liquidated users have to create a new account.. But not too sure about this